src/init.cpp

```diff
@@ -494,13 +493,17 @@ void SetupServerArgs()
     gArgs.AddArg("-sysperms", "Create new files with system default permissions, instead of umask 077 (only effective with disabled wallet functionality)", false, OptionsCategory::OPTIONS);
 #endif
     gArgs.AddArg("-version", "Print version and exit", false, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-powcachesize", strprintf("Set max pow cache size (number of pwo hashes) that keeping in memory (default: %d)", DEFAULT_POW_CACHE_SIZE), false, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-powmaxloadsize", strprintf("Set max pow cache load size (number of pwo hashes) that to be written to powcache.dat (default: %d)", DEFAULT_MAX_LOAD_SIZE), false, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-powcachevalidate", "Set to whether validate pow hashes that it get from the cache (default: %true). Use this option will slow down wallet synchronization.", false, OptionsCategory::OPTIONS);
```

Comment:
Suggest different wording:
Enable validation of pow hashes from the cache (default: %true). Use of this option will significantly slow down wallet synchronization.